components/waitwhile/app/waitwhile.app.ts

```diff
@@ -7,7 +9,69 @@ export default defineApp({
   methods: {
     // this.$auth contains connected account data
     authKeys() {
-      console.log(Object.keys(this.$auth));
+      console.log(this.$auth);
+    },
+    _authToken() {
+      return this.$auth.api_key;
+    },
+    _client() {
+      const sdk = api('@waitwhile/v2#5t18rl5qsy3p0');
+      sdk.auth(this._authToken());
+      return sdk;
+    },
+    async createCustomers(params: Object): Promise<any>{
```

Comment:
Make sure to run [npx eslint](https://pipedream.com/docs/components/guidelines/#prerequisites) on the files before pushing them. When I ran it on this file, I was getting errors for using type Object, but it worked fine when I changed Object to Record<string, unknown>.
```suggestion
    async createCustomers(params: Record<string, unknown>): Promise<any>{
```

Reply:
I'm still modifying the components with the changes listed above.